Dockerfile

```diff
@@ -117,6 +121,8 @@ COPY --link --from=buildctl /usr/bin/buildctl /buildctl.exe
 FROM binaries-$TARGETOS AS binaries
 
 FROM --platform=$BUILDPLATFORM alpine:${ALPINE_VERSION} AS releaser
+# enable scanning for this stage
+ARG BUILDKIT_SBOM_SCAN_STAGE=true
```

Comment:
I think this should be defined just in the `binaries` stage. `go build` stages also pick up lot of noise and other stages are added by default.